agent/utils/ticker_test.go

```diff
@@ -38,7 +44,7 @@ func TestTickerHappyCase(t *testing.T) {
 			break
 		}
 	}
-	if times < 8 || times > 100 {
+	if times < lowerTickLimit || times > upperTickLimit {
 		t.Error("Should tick at least 8 but less than 100 times: ", times)
```

Comment:
Maybe we should use constants in the error message as well?

Reply:
Done.